sklearn/ensemble/base.py

```diff
@@ -77,6 +77,9 @@ def _make_estimator(self, append=True):
         estimator.set_params(**dict((p, getattr(self, p))
                                     for p in self.estimator_params))
 
+        if random_state is not None:
+            randomize_estimator(estimator, random_state)
```

Comment:
You're right that we can implement it as `sklearn.ensemble.base._set_random_state` for now and get it merged without giving it a public name. And ignoring the lacks of `testing.set_random_state`. What do you think?